client/src/components/NoteManagementModal/NoteManagementModal.js

```diff
@@ -0,0 +1,35 @@
+import { MdClose } from "react-icons/md";
+import { NoteManagementForm } from "./components";
+import "./noteManagementModal.css";
+
+const NoteManagementModal = ({
+  note = { _id: "", title: "", content: "" },
+  onNoteSubmit,
+  onClose,
+}) => {
+  const handleSubmit = (noteData) => {
+    if (note._id) {
+      onNoteSubmit(note._id, noteData);
+    } else {
+      onNoteSubmit(noteData);
+    }
+    onClose();
+  };
+
+  return (
+    <div className="modal-overlay">
+      <div className="modal-content">
+        <button className="modal-close" onClick={onClose}>
+          <MdClose size={24} />
+        </button>
+        {note._id ? (
```

Comment:
Instead of doing this you can call the component directly as <NoteManagementForm note={note} onSubmit={handleSubmit} /> since in create even you have initial value but the id is empty